src/utils/jump_utils.jl

```diff
@@ -106,13 +112,14 @@ function _to_matrix(
 end
 
 function to_matrix(array::SparseAxisArray{T, N, K}) where {T, N, K <: NTuple{N, Any}}
+    # Don't use _get_column_names to avoid allocation of the columns vector
```

Comment:
It doesn't matter for correctness. I'm wondering if you want the columns sorted so that the results are deterministic for users.

Reply:
we then would have to call `sort!(unique!())` instead of set.

Reply:
226de8d